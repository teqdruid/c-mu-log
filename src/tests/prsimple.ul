foo(4,5);
@learn("$x");
foo(){OR:
@learn(wall(4,5););
//@forget( wall(2,3); );
}
