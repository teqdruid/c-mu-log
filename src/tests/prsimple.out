foo(4,5);
@learn("learn");
@learn(wall(4,5);wall(4);wall(););
